src/cplscheme/CouplingScheme.cpp

```diff
@@ -9,6 +9,7 @@ const int CouplingScheme::UNDEFINED_TIME_WINDOWS = -1;
 const double CouplingScheme::UNDEFINED_TIME_WINDOW_SIZE = -1.0;
 
 const int CouplingScheme::UNDEFINED_MAX_ITERATIONS = -1;
```

Comment:
Isn't the key for `UNDEFINED_MAX_ITERATIONS` now identical with the key for an infinite number of iterations? I think it would be good to define another specific `const int` for this purpose.

Reply:
No iterations given is interpreted as infinite iterations. So, conceptually nothing changed. It's only interpreted differently.